app/app.go

```diff
@@ -0,0 +1,70 @@
+package app
+
+import (
+	"database/sql"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+//nema pojma za db i web
```

Comment:
We should not keep comments like this in our source code. Some `NOTE: ...` comments are okay in case they're explaining why something has been done and the purpose, but in general try to clean the code up before finalizing the pull request (run the formatter, add or remove new lines here and there, remove comments, proper naming, etc.)